satnogsclient/web/static/js/upsat-control.js

```diff
@@ -338,23 +362,48 @@ $(document).ready(function() {
         } else if (selected_value == "adcs") {
             // TODO: Is app_id needed in time service?
             //app_id = $('#service-param-time-app_id').val();
-            app_id = 7;
-            type = 0;
-            ack = 0;
+            app_id = 3;
+            type = 1;
+            ack = $('#service-param-adcs-ack').val();
 
             service_type = 3;
             service_subtype = 23;
-            dest_id = 3;
+            dest_id = $('#service-param-adcs-dest_id').val();
 
             data = [];
-            ascii_to_dec($('#service-param-service-tle').val().split(''), data);
-            data.unshift(6);
-            //number of TLE chanacters
-            if (data.length != 137) {
-                alert("TLE shouldnt be: " + data.length);
-                return 0;
+            data.splice(0, 0, service_type);
```

Comment:
ok, as service type can be confusing with the ecss packet service type, can you change it with something like function_type?